### PR TITLE
edited document

### PR DESCRIPTION
**Pull request by platkis**
Edited the document, a couple things I wasn't sure about from the rubric:

- What is a roadmap?
- Top down vs. bottom up method for the hazard analysis technique
- Categorizing the hazards. I was thinking maybe major and minor but wasn't sure if you thought of a better way.

### CODE REVIEW

File: documentation/HazardAnalysis/HazardAnalysis.tex [hunk 1]
```diff
@@ -75,6 +75,7 @@
    \hline
    \textbf{Date} & \textbf{Revision} & \textbf{Comments} & \textbf{Author(s)} \\ \hline
    Dec 29, 2017 & 1.0 & Main content done for all sections & Christopher McDonald \\ \hline
+   Jan 1, 2018 & 2.0 & Edit entire document & Sharon Platkin \\ \hline
```
Comment (ChristopherMcDonald) [inline documentation/HazardAnalysis/HazardAnalysis.tex]: can we change to 1.1?

File: documentation/HazardAnalysis/HazardAnalysis.tex [hunk 2]
```diff
@@ -115,30 +117,30 @@ \subsection{Naming Conventions and Terminology}
...
    \includegraphics[width=0.7\textwidth]{img/ft-Air.png} % requires the graphicx package
    \caption{Fault Analysis Tree for \textit{User is struck by an air-borne ball}}
-   \label{fig:ft-Air}
+   \label{fig:ft-Air} %TODO: ITTFF should be ITTF
 \end{figure}
 
 \subsection{User is caught in a Pinch Point}
 \subsubsection*{Description}
-The system will include moving mechanisms to shoot balls toward the player in a variety of ways. This means that two pieces could be moving next to one another and a user's finger, or other extremity, could be caught in between the two pieces. This can cause damage to the user and the system. The system can break the users skin or bones and the motors and materials can break if given enough force.
+The system will include moving mechanisms to shoot balls toward the player in a variety of ways. This means that two pieces could be moving next to one another. This may lead to a user's finge or other extremity being caught in between the two pieces. This can cause damage to the user and the system. The system can break the users skin or bones and the motors and materials can break if given enough force.
```
Comment (ChristopherMcDonald) [inline documentation/HazardAnalysis/HazardAnalysis.tex]: finge => finger, being => getting

File: documentation/HazardAnalysis/HazardAnalysis.tex [hunk 3]
```diff
@@ -152,7 +154,7 @@ \subsection{Electric Shock}
 \subsubsection*{Description}
 The system will require electrical components in order to activate motors. This will also include power balancing and transfer to the necessary components. This means some wiring will be needed and thus some inherent risk of shock to the user via such wiring or the sensors or actuators themselves. Only when the wires carry current would this be a risk.
 \subsubsection*{Mitigation Plan}
-All wiring will be enclosed to the best of our ability. However, due to the nature of the system requiring motion it can be difficult to completely enclose the system. Proper warnings and notices will be used to warn the user of electrical current. The system should also have a fail-safe which cuts power if it short circuits. \\ \\
+All wiring will be enclosed to the best of our ability. However, due to the nature of the system requiring motion it can be difficult to completely enclose the system. Proper warnings and notices will be used to warn the user of electrical current. The system will also be fused to prevent prolonged short circuits. \\ \\
```
Comment (ChristopherMcDonald) [inline documentation/HazardAnalysis/HazardAnalysis.tex]: I would revert this change and say 'have a fail-safe via a fuse, which cuts power...'

Comment (platkis) [reply documentation/HazardAnalysis/HazardAnalysis.tex]: Not huge on the term "cut the power", can we use another phrasing?

Comment (ChristopherMcDonald) [reply documentation/HazardAnalysis/HazardAnalysis.tex]: Uhhh... maybe "cease power input"?

Comment (ChristopherMcDonald) [reply documentation/HazardAnalysis/HazardAnalysis.tex]: Lets get all the changes in asap though :)

**Comment by ChristopherMcDonald** [comment]
@raynerjs the hazard you added doesn't seem like it directly causes harm to user or assets, I did have that idea in other hazards like user being struck by the ball. Can we remove it / change it?

**Comment by raynerjs** [comment]
@ChristopherMcDonald yeah you're right, we can just remove it. I don't think it adds much that the other hazard doesn't.
